CHANGES.md

```diff
@@ -1,17 +1,33 @@
-# UNRELEASED
-- [FIXED] README to align with the 2.8.0 Kafka version.
-- [BREAKING CHANGE] Refactor package names
-- [BREAKING CHANGE] Source and sink connector moves and renames
-- [BREAKING CHANGE] Source connector flatten, schema generation and omit design documents options have been replaced by message transforms. See README for details.
-- [BREAKING CHANGE] Source connector now emits `java.util.Map` (not `String`) record values by default. See README for details.
-- [BREAKING CHANGE] Source connector now emits tombstone events for deleted documents. See [single message transforms](README.md#single-message-transforms) section in README for details.
+# 0.200.0 (2022-11-01)
+
+- [FIXED] README and documentation have been extensively rewritten.
+- [BREAKING CHANGE] Rename source connector. Properties files should be updated to
+  use `connector.class=com.ibm.cloud.cloudant.kafka.SourceChangesConnector`.
+- [BREAKING CHANGE] Rename sink connector. Properties files should be updated to
+  use `connector.class=com.ibm.cloud.cloudant.kafka.SinkConnector`.
+- [BREAKING CHANGE] Configuration parameters have changed for url, database, authentication, and last change sequence.
+  See README for details.
+- [BREAKING CHANGE] Source connector flatten, schema generation and omit design documents options have been replaced by
+  message transforms. See README for details.
+- [BREAKING CHANGE] Source connector now emits `java.util.Map` (not `String`) record values by default. See README for
+  details.
+- [BREAKING CHANGE] Source connector now emits `org.apache.kafka.connect.source.SourceRecord` (not `String`) record keys
+  by default. See README for details.
```

Comment:
A little unsure about "by default" here, I guess valid in the sense that it can be transformed, but we don't provide any transforms for it like we do for the event values (where I guess this is copied from).